debugtools/DDR_VM/src/com/ibm/j9ddr/AuxFieldInfo29.dat

```diff
@@ -266,6 +266,7 @@ J9JavaVM.jitConfig = required
 J9JavaVM.jniGlobalReferences = required
 J9JavaVM.jniWeakGlobalReferences = required
 J9JavaVM.jvmtiData = required
+J9JavaVM.liveVirtualThreadList = J9Object**
```

Comment:
This should say `j9object_t*` to match the declaration in `j9nonbuilder.h`.

Reply:
Apparently, I gave you bad advice:
```
Unhandled structure type: J9JavaVMPointer->liveVirtualThreadList j9object_t*
```
it will need to change back to
```
J9JavaVM.liveVirtualThreadList = J9Object**
```